assets/app/view/game/cube_chart.rb

```diff
@@ -0,0 +1,124 @@
+# frozen_string_literal: true
+
+# attrs frozen_string_literal: true
+
+require 'view/game/actionable'
+require 'lib/settings'
+
+module View
+  module Game
+    class CubeChart < Snabberb::Component
+      include Actionable
+      include Lib::Settings
+
+      PAD = 4                                     # between box contents and border
+      BORDER = 1
+      WIDTH_TOTAL = 50                            # of entire box, including border
+      HEIGHT_TOTAL = 50
+
+      def render
+        @cube_chart = @game.cube_chart
+
+        @box_style = {
+          position: 'relative',
+          display: 'inline-block',
+          padding: "#{PAD}px",
+          width: "#{WIDTH_TOTAL - (2 * PAD) - (2 * BORDER)}px",
```

Comment:
Why do you need so much control over the dimensions? Grid and flex are designed to work seamlessly.